modules/ROOT/pages/mule-app-tutorial.adoc

```diff
@@ -70,25 +70,25 @@ image::dw-quickstart-add-db2.png[]
 ** **Password**: `mule`
 ** **Database**: `training`
 +
-image::dw-quickstart-db-values.png[]
+image::mruntime-quickstart-db-values.png[]
 . While still in the Database Config dialog, select **Configure > Add Maven dependency**.
 . In the **Pick a Maven dependency** dialog, enter `mysql-` in the **Search Maven Central** search field.
 . Select `mysql:mysql-connector-java` from the displayed items.
 . Select **Finish** to return to the Database Config dialog.
 +
-image::dw-quickstart-maven1.png[]
+image::mruntime-quickstart-db-driver.png[]
 . Select **Test Connection**. You should receive a `Test connection successful` message. If you don't, go back through the steps and look for errors.
 . Select **OK** to return to the Database Config dialog.
 . Create a query that returns all flights from the Training: American Flights API.
 .. If it's not already open, click the Select operation to display its property tab.
 .. Add a select statement in the query field: `SELECT * FROM american`. Your changes are automatically saved.
 +
-image::dw-quickstart-select-query.png[]
+image::mruntime-quickstart-select-query.png[]
 . To test your configuration so far, run the project. You can right-click in the canvas where the flow is defined, and select **Run Project dw-tutorial4-flights-ws**.
 . Watch the Console tab and when the app is running, open your REST API client.
-. In the client, send a request to http://localhost:8081/flights/[http://localhost:8081/flights/]. You should see a Mule object in the result.
+. In the client, send a request to `+http://localhost:8081/flights/+`. At this point, the application returns a `500 Server Error` message because it cannot process the data received from the query to the American Flights API. This error is addressed in the next step when the Transform component is added to the flow.
```

Comment:
```suggestion
. In the client, send a request to `+http://localhost:8081/flights/+`. 
This request returns a `500 Server Error` message because the application cannot process the data received from the query to the American Flights API. This error is addressed in the next step when the Transform component is added to the flow.
```
If the following step resolves this error, consider omitting the final sentence in favor of that step, maybe something like ". Resolve the server error by adding the Transform component to the flow:" maybe with substeps? I don't want to look ahead, because it might endanger my health...lol....